js/main.js

```diff
@@ -9,10 +9,15 @@ const $form = document.querySelector('#new-entry');
 
 $form.addEventListener('submit', function (event) {
   event.preventDefault();
-  const newEntry = {};
+  const newEntry = {
+    title: null,
+    photoURL: null,
+    notes: null,
+    entryId: null
```

Comment:
These shouldn't be `null`. You should be storing the `input`s values from your `form`